policy-controller/core/src/lib.rs

```diff
@@ -108,9 +108,3 @@ impl Ord for InboundHttpRouteRef {
         }
     }
 }
-
-impl PartialOrd for InboundHttpRouteRef {
-    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
-        Some(self.cmp(other))
-    }
-}
```

Comment:
clippy would like us to put the `PartialOrd` impl back: https://github.com/linkerd/linkerd2/runs/7632344017?check_suite_focus=true#step:5:200

Reply:
Yea I didn't know clippy would complain about that. It was already added back in [3a34f97](https://github.com/linkerd/linkerd2/pull/9008/commits/3a34f9734a79c159d98660cf9098df97b13bcbd2) though!